docs/concepts/collection.md

```diff
@@ -0,0 +1,29 @@
+# Collections
+
+Collections are groups of related content that can be viewed or managed as a unit.  Islandora-specific use cases include:
+
+- an archival fonds that needs to be grouped together, with internal hierarchy
+- various collections of artifacts, grouped for display
+- theses and dissertations, which are organized and managed separately from other objects.
+
+Islandora can:
+
+- declare items as members of other objects
+- after declaring an item "a collection", display a view of its members
+
+Management of the objects of a collection is not yet readily built in, but such actions as deleting, changing visibility or permissioning on a collection should be possible through Views Bulk Edit and other contrib modules, such as Permissions By Term.
```

Comment:
Suggestion (although I still don't know if I fully understand what Defaults actually _is_): "A minimal installation of Islandora (Islandora Core) does not provide pre-configured options to perform management operations on all objects of a collection. Bulk actions such as deleting, changing visibility or adjusting permissions on all objects in a collection can be achieved with the _Views Bulk Edit_ Drupal module (link) and other Drupal contrib modules, such as _Permissions By Term_ (link). The Islandora Defaults feature package (???) includes a model configuration and tools to manage objects in collections."

Reply:
"The Islandora Defaults feature package (???) includes a model configuration and tools to manage objects in collections."" -- i dont know,  let's build a Defaults to find out! (Thanks @dannylamb for your help on this!)

Reply:
Side-note, I'd rather not recommend permissions_by_term due to performance degradation as repository size grows. I would feel better if we dropped ", such as Permissions By Term".